dom.bs

```diff
@@ -8705,6 +8705,25 @@ associated <a for="/">element</a>'s
   <p>Throws a <code>TypeError</code> exception if the associated attribute has no supported tokens defined.
 </dl>
 
+<p>The <a method for=DOMTokenList lt="supports()">supports()</a> method can be used to detect support for {{DOMTokenList}} tokens:
+
+<pre class=lang-javascript>
+var DOMTokenListSupports = function(tokenList, token) {
+    if (!tokenList || !tokenList.supports) {
+        return;
+    }
+    try {
+        return tokenList.supports(token);
+    } catch (e) {
+        console.log("The DOMTokenList doesn't have a supported tokens list");
+    }
+};
+
+var linkSupportsToken = DOMTokenListSupports(document.createElement("link").relList, supportedToken);
+var classListSupportsFoo = DOMTokenListSupports(document.createElement("div").classList, "foo"); // undefined + console.log
+var linkSupportsFoo = DOMTokenListSupports(document.createElement("link").relList, "foo"); // false
```

Comment:
I don't see a need for two examples for link. JavaScript developers know how to use variables.